### PR TITLE
Add package.json to allow adding dependencies

### PR DESCRIPTION
**Pull request by micgro42**
This should unlock of the follow-up tasks

Bug: T300883

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wmde/new-lexeme-special-page.git"
+  },
+  "keywords": [
+    "Wikidata"
+  ],
+  "author": "The Wikidata team",
+  "license": "GPL-2.0-only",
```
Comment (lucaswerkmeister) [inline package.json:18]: Should it really be GPL2 *only*? I thought the default is or-later.

Comment (micgro42) [reply package.json:18]: Mh, our internal guideline doesn't say. I'll look at what we're using in other places.

Comment (micgro42) [reply package.json:18]: We don't seem to have this field in most `package.json` files. But the first WikibaeLexeme php file that I opened had a license tag for `GPL-2.0-or-later`, so I'm using that here as well

Comment (micgro42) [reply package.json]: Done.

**Review by lucaswerkmeister — APPROVED** [review]

**Pull request merged**